ts/Stock/Indicators/EMA/EMAIndicator.ts

```diff
@@ -66,7 +66,7 @@ class EMAIndicator extends SMAIndicator {
      * @extends      plotOptions.sma
      * @since        6.0.0
      * @product      highstock
-     * @requires     stock/indicators/indicators
+     * @requires     indicators/indicators
```

Comment:
Is this the correct path? Shouldn't we include `stock`? So the URL in [API ref ](https://api.highcharts.com/highstock/series.ema) will also be correct? 

For example: https://code.highcharts.com/indicators/indicators.js - 404

The same applies to all other indicators

Reply:
Oh, you're right! I didn't think of that.